4_turb_abl_rigid/run-frontier.sh

```
@@ -0,0 +1,60 @@
+#!/bin/bash -l
+
+#SBATCH -J 4_turb_abl_rigid
+#SBATCH -o %x.o%j
+#SBATCH -A CFD162
+#SBATCH -t 30
+#SBATCH -S 0
+#SBATCH -N 64
+
```

why remove `set -e`?

---

In my testing, if `set -e` is enabled, then executing `module load amd-mixed/6.1.3` causes the shell to exit immediately. However, I needed to load this module to get exawind to run. So, my solution was to remove `set -e` from the top of the run scripts. However, if this is important, perhaps your `spack build-env <package>` solution below will be a good work around. Either that or we could `set -e` after loading the modules. I'll do some further tests with both solutions.

---

Makes sense. Maybe just move the `set -e` after the module load? I guess we could think more deeply about whether it matters for these batch scripts, in general I like it because if one thing (e.g. activating the spack environment) fails we usually want to exit right away instead of continuing to try to run.